src/containers/profile/settingsTab/styles.module.scss

```diff
@@ -0,0 +1,94 @@
+@import '../../../scss/shared/breakpoints.scss';
+@import '../../../scss/shared/fonts.scss';
+@import '../../../scss/shared/colors.scss';
+@import '../../../scss/mixins/button.scss';
+@import '../../../scss/mixins/input.scss';
+
+.settings {
+  margin-top: 40px;
+  .localization{
+    width: 100%;
+    select{
+      width: 250px;
+      margin: 15px 5px;
+      background-color: transparent;
+      padding: 15px;
+      border: 3px #212121 solid;
+      color: #fff;
+      cursor: pointer;
+      option{
+        background-color: #262626;
+        cursor: pointer;
+      }
+    }
+  }
+  ul{
```

Comment:
codestyle: do we want a space before curly bracket? Try to pick one and stick to it.